stable/jenkins/templates/deprecation.yaml

```diff
@@ -9,8 +9,12 @@
            {{ fail "`Master.Image` does no longer exist. It has been renamed to `master.image`" }}
         {{- end }}
 
+        {{- if .Values.master.imageTag }}
+           {{ fail "`master.imageTag` will be obsoleted soon. It has been renamed to `master.tag`" }}
```

Comment:
fail results in an error. So the chart can not be deployed before fixing it.

I suggest to move the if statement to the NOTES.txt and print a warning instead.